beacon-chain/core/blocks/payload.go

```diff
@@ -23,6 +23,8 @@ import (
 // Spec code:
 // def is_merge_transition_complete(state: BeaconState) -> bool:
 //    return state.latest_execution_payload_header != ExecutionPayloadHeader()
+//
+// DEPRECATED: Use `IsMergeTransitionBlockUsingPayloadHeader` instead.
```

Comment:
```suggestion
// Deprecated: Use `IsMergeTransitionBlockUsingPayloadHeader` instead.
```

I'm not 100% sure if the capitalization matters. Did your IDE highlight this method as deprecated in all caps? 
I have been having issues with IDE strikethrough for deprecation warnings.